libs/application/templates/estate/src/forms/ResidencePermit/overviewSection.ts

```diff
@@ -54,10 +58,12 @@ export const overview = buildSection({
           {
             cards: ({ answers }: Application) =>
               ((answers.estate as any).estateMembers ?? []).map(
-                (member: EstateMember) => ({
+                (member: EstateMember | any) => ({
```

Comment:
Isn't `EstateMember | any` just `any`?

Reply:
Is there something missing from the EstateMember type?
Consider adding it if so?